src/components/Dashboard/Form/ChildData/ChildData.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import FieldWrapper from "../FieldWrapper/FieldWrapper";
+import styles from "./ChildData.scss";
+
+const ChildData = () => (
+  <div className={`FormContent ${styles.ChildData}`}>
+    <FieldWrapper name={`child.name`} componentSize="large" component="input" />
+    <FieldWrapper name={`child.name`} componentSize="large" component="input" />
+    <FieldWrapper
+      name={`child.birthPlace`}
+      componentSize="large"
+      component="input"
+    />
+    <FieldWrapper
+      name={`child.name`}
+      componentSize="medium"
+      component="input"
+    />
+    <FieldWrapper
+      name={`child.pesel`}
+      componentSize="large"
+      component="input"
+    />
+    <FieldWrapper name={`city`} componentSize="medium" component="input" />
+    <FieldWrapper name={`city`} componentSize="large" component="input" />
+    <FieldWrapper name={`city`} componentSize="medium" component="input" />
+    <FieldWrapper name={`city`} componentSize="large" component="select" />
```

Comment:
4x city?

Reply:
yep, because we don't have postalCode, steet etc. so I filled it with something related to city from fixture.js